komodoenv/komodo.py

```diff
@@ -1,15 +0,0 @@
-from subprocess import Popen, PIPE
-
-
-def get_komodoenv_version(root):
```

Comment:
I couldn't find this in use - should I? Not sure i fully understand what this code is supposed to do if it should stay ?

Reply:
I think it got in by accident. We use a different method to obtain the komodo version currently.